internal/logger/logger.go

```diff
@@ -38,6 +39,7 @@ func NewLogger(h ...slog.Handler) *slog.Logger {
 	} else {
 		handler = slog.NewJSONHandler(os.Stderr, &slog.HandlerOptions{
 			AddSource: true,
+			Level:     getLevel(os.Getenv("LOG_LEVEL")),
```

Comment:
I don't know if we really need to practically refined the log levels with our own getLevel function, as slog seems to handle that already. If someone passes a wrong log level or an empty one, slog will default to INFO.

Reply:
Really I didn't find anything about that?

Maybe I just didn't read the docs enough 😅

Reply:
I'm sorry, that was a misunderstanding on my part. It's fine that way.